### ISSUE TITLE
Odd error message, concerning numpy using an alias versus numpy package name

### ISSUE THREAD

**Issue by apiszcz** [pandas-dev/pandas#28665]
This produces a poor warning

```python
>>> import pandas as pd
>>> import numpy as np
>>> np.ptp(pd.Series([1, 2, 3]))
/lib/python3.7/site-packages/numpy/core/fromnumeric.py:2495: FutureWarning: Method .ptp is deprecated and will be removed in a future version. Use numpy.ptp instead.
```

because `np.ptp` calls `Series.ptp`, which is deprecated. We should instruct the user
to convert the Series to an array before calling `ptp`.

```python
np.ptp(pd.Series([1, 2, 3]).array)
```

The warning is defined at https://github.com/pandas-dev/pandas/blob/20e4c186191859dcde2437edacaee43d8d34dc46/pandas/core/generic.py#L10197-L10199

---

In this case I am using numpy ptp, however the error messages states I am not

```
>>> dfagg = df.groupby(['idf']).agg({ 'f1' : [np.min, np.max, np.ptp] })
C:\test\lib\numpy\core\fromnumeric.py:2495: FutureWarning: Method .ptp is deprecated and will be removed in a future version. Use numpy.ptp instead.
```

NO warning
`>>> dfagg = df.groupby(['idf']).agg({ 'f1' : [np.min, np.max, numpy.ptp] })`

**Comment by DavidMStraub** [comment]
I'm also running into this warning. It's actually even simpler to reproduce:

```python
import pandas as pd
import numpy as np
np.ptp(pd.Series([1, 2, 3]))
```
This will raise the warning. The reason, I think, is ironic: `pandas` warns against using the Series' method inspite of the function, but `numpy`, when using the function, tries if there is a method and uses that preferably:
https://github.com/numpy/numpy/blob/master/numpy/core/fromnumeric.py#L2541-L2548

**Comment by TomAugspurger** [comment]
We faced a similar issue with argmin / argmax: https://github.com/pandas-dev/pandas/pull/16955/files

The recommendation is to have people do `np.ptp(pd.Series([1, 2, 3]).to_numpy())` in the meantime. Does anyone want to update the deprecation message?

**Comment by hasnain2808** [comment]
I would like to take up this issue but as I am new would need some support

**Comment by hasnain2808** [comment]
take

**Comment by jbrockmendel** [comment]
closed by #30458.
